src/components/Details/VmTemplateDetails/VmTemplateDetails.js

```diff
@@ -124,10 +143,10 @@ export class VmTemplateDetails extends React.Component {
   isFormValid = () => Object.keys(this.state.form).every(key => this.state.form[key].valid);
 
   render() {
-    const { vmTemplate, dataVolumes, NamespaceResourceLink, LoadingComponent, k8sGet } = this.props;
+    const { vmTemplate, dataVolumes, NamespaceResourceLink, LoadingComponent } = this.props;
     const id = getId(vmTemplate);
     const vm = selectVm(vmTemplate.objects);
-    const baseTemplate = getVmTemplate(vmTemplate);
+    const baseTemplateLabels = getTemplateDisplayName(getVmTemplate(vmTemplate));
```

Comment:
nit: variable name

Reply:
sorry ... `baseTemplateName` ?

Reply:
np, yup could be or baseTemplateDisplayName